File: C++/terreno.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main() {
    cout << fixed << setprecision(1);
    cout << "Qual o valor da largura do retangulo? ";
    double largura;
    cin >> largura;
    cout << "Qual o valor da altura do retangulo? ";
    double altura;
    cin >> altura;
    double area = largura * altura;
    cout <<"Digite o valor do metro quadrado: ";
    cout << fixed << setprecision(2);
    double valorMetroQuadrado;
    cin >> valorMetroQuadrado;
    double valorTerreno = area * valorMetroQuadrado;
    cout << "A area do terreno é: " << area << endl;
    cout << "O valor do terreno é: " << valorTerreno << endl;
    return 0;
}

File: Terreno.cs
using System;
					
public class Program
{
	public static void Main()
	{
		float c, l, a, v, p;
		Console.WriteLine("Digite o comprimento do terreno: ");
		c = float.Parse(Console.ReadLine());
		Console.WriteLine("Digite a largura do terreno: ");
		l = float.Parse(Console.ReadLine());
		a = c*l;
		Console.WriteLine("Digite o valor do metro quadrado do terreno: ");
		v = float.Parse(Console.ReadLine());
    p = v*a;
    Console.WriteLine("A area do terreno é " +a.ToString("F2));
    Console.WriteLine("O valor do preco do terreno é " +p.ToString("F2"));
		
	}
}
